.condarc

```diff
@@ -0,0 +1,6 @@
+ssl_verify: False
+channels:
+  - zeitsperre
+  - cjauvin
+  - conda-forge
+  - defaults
```

Comment:
Do you mind adding newlines to the end of your files? I think in PyCharm you can do this:
https://stackoverflow.com/a/37260222/787842